### DISCUSSION TITLE
add creating items and selecting items interface options for m2o as well

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#6012] [Feature Requests]
like in the `o2m` interface:

![image](https://user-images.githubusercontent.com/10562635/120547130-f8e11500-c3f0-11eb-8847-33741bb43d60.png)

plus I'd add the option to **not** enable the remove button to both `o2m`/`m2o`

**Comment by benhaynes** [comment]
Duplicate: https://github.com/directus/directus/discussions/2742

Also, this says implemented, so it should already be on the latest release!

**Reply by rijkvanzanten** [reply]
@benhaynes That linked discussion is to add support for the Add New button in general, but doesn't mention a toggle to then _disable_ it 👍🏻
